server/src/main/java/org/apache/uniffle/server/storage/MultiStorageManager.java

```diff
@@ -96,7 +96,15 @@ private StorageManager selectStorageManager(ShuffleDataFlushEvent event) {
     if (event.getSize() > flushColdStorageThresholdSize) {
       return coldStorageManager;
     } else {
-      return warmStorageManager;
+      try {
```

Comment:
What will happen when both the cold storage manager and warm storage manager are broken? It seems to fall back many times.